### ISSUE TITLE
Unable to replicate experiments for 80 epochs

### ISSUE THREAD

**Issue by snailrowen1337** [sthalles/PyTorch-BYOL#4]
Thanks a ton for making this open-source. I have a question about replicating the 75% accuracy for 80 epochs. I just changed the config file to read "max_epochs: 80" and then trained the model, but it only reached roughly 70% accuracy, see below. Is there anything else that needs to be changed to reach the 75% accuracy you report? Thanks!!

Testing accuracy: 69.5125
Testing accuracy: 69.5125
Testing accuracy: 69.6875
Testing accuracy: 69.9125
Testing accuracy: 69.7875
Testing accuracy: 69.7
Testing accuracy: 69.625
Testing accuracy: 69.8625
Testing accuracy: 69.65
Testing accuracy: 69.5375
Testing accuracy: 70.025
Testing accuracy: 69.8125

**Comment by BrettLL** [comment]
@snailrowen1337 my test acc is always: 10.  no matter which epoch I used.

**Comment by soumye** [comment]
@snailrowen1337  I had the same issue. Did you find any fix for this problem?
